packages/mux-player/src/themes/gerwig/gerwig.html

```diff
@@ -132,40 +132,33 @@
       transform: translate(-50%, -10px);
     }
 
-    :is(media-captions-listbox, media-rendition-listbox, media-audio-track-listbox, media-playback-rate-listbox)::part(
-        indicator
-      ) {
+    [part~='listbox'] {
+      border-radius: 2px;
+      border: 1px solid rgba(0, 0, 0, 0.1);
+      bottom: calc(100% + 16px);
+      padding: 2.5px 10px;
```

Comment:
the internal container already had 7.5px vertical padding so just add 2.5px here for 10px like in the design